spec/runners/unified/test.rb

```diff
@@ -156,6 +156,15 @@ def create_entities
                 end
               end
             end
+            if @observe_secnsitive = spec.use('observeSensitiveCommands')
+              %w(authenticate saslStart saslContinue getnonce createUser updateUser copydbgetnonce copydbsaslstart copydb hello ismaster isMaster).each do |event|
+                  subscriber = (@subscribers[client] ||= EventSubscriber.new)
+                  unless client.send(:monitoring).subscribers[Mongo::Monitoring::COMMAND].include?(subscriber)
+                    client.subscribe(Mongo::Monitoring::COMMAND, subscriber)
+                  end
+                  subscriber.add_wanted_events(event)
```

Comment:
This method is used to indicate which events are wanted (e.g. CommandStarted), not which commands should be recorded. I am confused what this call is actually doing/is meant to do?

Reply:
Thanks, good point. Removed.